### PR TITLE
ML Resources

### PR DESCRIPTION
**Pull request by king-11**
- update nav data add ml link
- page for ml resources
- data preprocessing links
- ensemble models links
- basics of nlp links

### CODE REVIEW

**Comment by king-11** [comment]
draft pr need to add nlp in ds resources till then @nishantkr18 please suggest any changes

File: _data/nav.yml [hunk 1]
```diff
@@ -57,6 +57,13 @@
       page: /resources/dev/
       tpl: dev.html
 
+    - id: resources.ml
+      i18n: resources.ml
+      name: ML
+      ref: resources.dev
```
Comment (nishantkr18) [inline _data/nav.yml]: shouldn't this be `resources.ml`? :thinking:

Comment (king-11) [reply _data/nav.yml]: ohh hindsight will correct it

**Review by nishantwrp — APPROVED** [review]
Just ping me when to merge this pr

File: _data/nav.yml (hunk 1 above)
Comment (nishantkr18) [reply _data/nav.yml]: @king-11 updates?

Comment (king-11) [reply _data/nav.yml]: @nishantkr18  updated

**Pull request merged**